### DISCUSSION TITLE
Panda extracting styles from non-panda component props

### DISCUSSION THREAD

**Discussion by nassimbenkirane** [chakra-ui/panda#1668] [Q&A] [answered]
### Description

When I use panda with other component libraries that have the same props as panda, I expect generated styles from panda to only contain styles from panda components.
Instead, other components are detected in the AST and styles are generated for non panda components.

### Link to Reproduction

https://play.panda-css.com/7-DyunsmOs

### Steps to reproduce

1. See that there's a non panda component
<img width="337" alt="Capture d’écran 2023-11-10 à 09 52 34" src="https://github.com/chakra-ui/panda/assets/1640952/44ee96b1-af01-4d57-80fe-23050427f066">

2. Click on AST
3. See that jsx detected the props
<img width="213" alt="Capture d’écran 2023-11-10 à 09 53 11" src="https://github.com/chakra-ui/panda/assets/1640952/44296704-6d7b-4418-866f-eaf5ce342bcb">

4. Click on CSS
5. See that the CSS was generated for the non panda component
<img width="288" alt="Capture d’écran 2023-11-10 à 09 53 45" src="https://github.com/chakra-ui/panda/assets/1640952/014430d1-37ea-4d34-abd5-f62023cb018f">

### JS Framework

React (TS)

### Panda CSS Version

0.18.1

### Browser

Chrome 119.0.6045.105

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

In our case, we want to migrate to Panda incrementally, from a codebase that uses rebass + styled components + styled-system that has a similar API than panda 
```jsx
<Box mt='10px'></Box>
```

So we will have a lot of styles generated for the old code which is concerning

**Comment by astahmer** [comment, marked as answer]
due to static limitations, we can't know which of your custom components are using Panda internally or not
2 possible workaround:
- change your [`config.include`](https://panda-css.com/docs/references/config#include) to only include folders with Panda components
- change [`config.jsxStyleProps`](https://panda-css.com/docs/references/config#jsxstyleprops) to `minimal` so that only the `css` prop will be extracted from Panda
